djangocms_contentexpiry/models.py

```diff
@@ -0,0 +1,29 @@
+from datetime import datetime, timedelta
+
+from django.conf import settings
+from django.contrib.contenttypes.fields import GenericForeignKey
+from django.contrib.contenttypes.models import ContentType
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+
+class ContentExpiry(models.Model):
+    activation_date = models.DateTimeField(auto_now_add=True)
```

Comment:
Why is it called `activation_date`?

Reply:
No idea, It was in the story, basically its creation date of expiry object.